pos_partner_no_create/static/src/xml/templates.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates id="template">
+
+    <t t-extend="ClientListScreenWidget">
+        <t t-jquery='.new-customer' t-operation='replace'>
```

Comment:
Hi. Maybe add an invisible attribute, instead of replace, to avoid error if other module request ".new-customer".
don't you think.

Reply:
You're right, it's fixed